<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DayWindow</class>
 <widget class="QDialog" name="DayWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>451</width>
    <height>562</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="mainLayout">
     <item>
      <layout class="QHBoxLayout" name="topbarLayout">
       <item>
        <widget class="QLabel" name="topbarLabel">
         <property name="font">
          <font>
           <pointsize>14</pointsize>
          </font>
         </property>
         <property name="text">
          <string>You have no events on this day.</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="topbarSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="newEventButton">
         <property name="text">
          <string>New Event</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="eventsLayout"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
